src/app/ApplicationTemplate.Mobile/ApplicationTemplate.Mobile.csproj

```diff
@@ -11,6 +11,7 @@
 		<SupportedOSPlatformVersion Condition="'$(TargetFramework)' == 'net6.0-ios'">14.2</SupportedOSPlatformVersion>
 	</PropertyGroup>
 	<ItemGroup>
+		<PackageReference Include="Chinook.DataLoader.Uno.WinUI" Version="1.1.0" />
```

Comment:
Why was this added on this PR ? Were we missing it ?

Reply:
It was the Uno (UWP) before on the Xamarin heads projects. For .NET 6 we use Uno.WinUI variant.